arbitrum-docs/launch-orbit-chain/troubleshooting-building-orbit.md

```diff
@@ -7,7 +7,7 @@ content_type: faq
 ---
 
 import FAQStructuredDataJsonLd from '@site/src/components/FAQStructuredData';
-import FAQQuestions from '../partials/_troubleshooting-orbit-partial.md';
+import FAQQuestions from '../partials/_troubleshooting-orbit-partial.mdx';
```

Comment:
These are autogenerated files in `notion-docs/scripts/update.ts` so you'll have to modify the filenames there as well.

Reply:
Thank you, @TucksonDev, fixed in https://github.com/OffchainLabs/arbitrum-docs/pull/1456